requirements.txt

```diff
@@ -0,0 +1,7 @@
+kenlm
```

Comment:
whoops.  bad fingers.  the package is actually [pypi-kenlm](https://pypi.python.org/pypi/pypi-kenlm/0.1.20160618), and i only included it it because of #444 , which i also encountered. 

I missed the actual requirements.txt file, so mea cupla there as well.